src/other/Table.js

```diff
@@ -73,12 +72,34 @@
         ;
         return this;
     };
-
+    
     Table.prototype.publish("renderHtmlDataCells", false, "boolean", "enable or disable HTML within cells",null,{tags:["Private"]});
     Table.prototype.publish("pagination", false, "boolean", "Enable or disable pagination",null,{tags:["Private"]});
     Table.prototype.publish("fixedHeader", false, "boolean", "Enable or disable fixed table header and first column",null,{tags:["Private"]});
     Table.prototype.publish("fixedColumn", false, "boolean", "Enable or disable fixed first column",null,{tags:["Private"]});
-    Table.prototype.publish("autoCalcIPP", true, "boolean", "Enable or disable auto calculation of items per page",null,{tags:["Basic"]});
+    
+    Table.prototype.publish("theadFontSize", null, "string", "Table head font size", null, { tags: ["Basic"] });
+    Table.prototype.publish("tbodyFontSize", null, "string", "Table body font size", null, { tags: ["Basic"] });
+    Table.prototype.publish("theadFontColor", null, "html-color", "Table head font color", null, { tags: ["Basic"] });
+    Table.prototype.publish("tbodyFontColor", null, "html-color", "Table body font color", null, { tags: ["Basic"] });
+    Table.prototype.publish("theadFontFamily", null, "string", "Table head font family", null, { tags: ["Basic"] });
+    Table.prototype.publish("tbodyFontFamily", null, "string", "Table body font family", null, { tags: ["Basic"] });
+    
+    Table.prototype.publish("theadCellBorderColor", null, "html-color", "Table head cell border color", null, { tags: ["Basic"] });
+    Table.prototype.publish("theadRowBackgroundColor", null, "html-color", "Table head row color", null, { tags: ["Basic"] });
+    
+    Table.prototype.publish("tbodyCellBorderColor", null, "html-color", "Table body cell border color", null, { tags: ["Basic"] });
+    
+    Table.prototype.publish("tbodyRowBackgroundColor", null, "html-color", "Table body row color", null, { tags: ["Basic"] });
+    Table.prototype.publish("tbodyFirstColFontColor", null, "html-color", "Table body first column font color", null, { tags: ["Basic"] });
+    Table.prototype.publish("tbodyFirstColBackgroundColor", null, "html-color", "Table body first column background color", null, { tags: ["Basic"] });
+    
+    Table.prototype.publish("tbodyHoverRowFontColor", "#FFF", "html-color", "Table body hover row font color", null, { tags: ["Basic"] });
+    Table.prototype.publish("tbodyHoverRowBackgroundColor", null, "html-color", "Table body hover row background color", null, { tags: ["Basic"] });
+    
+    Table.prototype.publish("tbodySelectedRowFontColor", "#FFF", "html-color", "Table body selected row color", null, { tags: ["Basic"] });
+    Table.prototype.publish("tbodySelectedRowBackgroundColor", "#F48A00", "html-color", "Table body selected row color", null, { tags: ["Basic"] });
+    
```

Comment:
1.  I am not sure these are all Basic?
2.  This approach will break existing CSS driven changes (which may be broke anyway) - if you look at https://github.com/hpcc-systems/Visualization/pull/832/files I have added "optional" support for publish params, with that you can default to null for each of these which will use the default CSS and then you can change them at the dermatology level (or the CSS level as appropriate.